.gitignore

```diff
@@ -102,7 +103,10 @@ Tools/unicode/data/
 /config.log
 /config.status
 /config.status.lineno
+/m4/
```

Comment:
m4 is tracked directory. Why ignore it?
https://github.com/python/cpython/tree/master/m4

Reply:
Maybe I added it by mistake, but I can't confirm you, since I can't compile the latest trunk.